spec/requests/v0/user_spec.rb

```diff
@@ -11,6 +11,7 @@
     let(:mhv_user) { build(:user, :mhv) }
     let(:v0_user_request_headers) { {} }
     let(:edipi) { '1005127153' }
+    let!(:mhv_user_verification) { create(:mhv_user_verification, backing_idme_uuid: mhv_user.mhv_correlation_id) }
```

Comment:
oh sorry, I think I screwed up this suggested edit, it should be `mhv_uuid: mhv_user.mhv_correlation_id` here

Reply:
not `backing_idme_uuid`

Reply:
IT WORKS.  omg